libs/net/entities/Models/ContentVersion.cs

```diff
@@ -82,6 +86,7 @@ public ContentVersion(Content content, User owner)
         this.Body = content.Body;
         this.SourceUrl = content.SourceUrl;
         this.IsPrivate = content.IsPrivate;
+        this.Tone = content.TonePools.Count > 0 ? (int?)content.TonePools[0].Id : null;
```

Comment:
This incorrect, it should be the `TonePoolsManyToMany.FirstOrDefault().Value` not the `TonePool[0].Id`.